File: Muturi02/fizz_buzz.go
package main
import "fmt"
func main(){
	for i:=1;i<= 100;i++{
		if i%3==0 && i%5==0{//check this condition first. Output will be different if it is checked last. No "FizzBuzz" will be printed
			fmt.Print("FizzBuzz\n")
		}else if i %3==0{
			fmt.Print("Fizz\t")
		}else if i%5==0{
			fmt.Print("Buzz\t")
		}else{
			fmt.Print(i,"\t")
		}
	}
	fmt.Println()
}

File: BevNdombi15/fizzbuzz.py
def main():
	for i in range(1,101):
		is_multiple_of_three = i % 3 == 0
		is_multiple_of_five = i % 5 == 0
		if is_multiple_of_three and is_multiple_of_five:
			print("FizzBuzz")
		elif is_multiple_of_three:
			print("Fizz")
		elif is_multiple_of_five:
			print("Buzz")
		else:
			print(i)

if __name__ == '__main__':
	main()